go/obscuronode/host/host.go

```diff
@@ -17,32 +17,18 @@ import (
 	"github.com/obscuronet/obscuro-playground/go/obscuronode/wallet"
 )
 
-// P2PCallback -the glue between the P2p layer and the node. Notifies the node when rollups and transactions are received from peers
-type P2PCallback interface {
-	ReceiveRollup(r obscurocommon.EncodedRollup)
-	ReceiveTx(tx nodecommon.EncryptedTx)
-}
-
-// P2P is the layer responsible for sending and receiving messages to Obscuro network peers.
-type P2P interface {
-	StartListening(callback P2PCallback)
-	StopListening()
-	BroadcastRollup(r obscurocommon.EncodedRollup)
-	BroadcastTx(tx nodecommon.EncryptedTx)
-}
-
-// ClientServer is the layer responsible for handling requests from Obscuro client applications.
-type ClientServer interface {
-	Start()
-	Stop()
-}
+const ClientRPCTimeoutSecs = 5
```

Comment:
@otherview You should use the value from the host's config for this.

Reply:
Merge issue. Fixed 👍